Pages/Authentication/login.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+    margin: 0px;
+}
+
+:root {
+    --primary-color: #2563EB;
+    --gold: rgb(253, 196, 52);
+    --white: #ffff;
+    --black: rgba(0, 0, 0, 0.8);
+}
+
+.main-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 5rem;
+}
+
+.login-container {
+    width: 350px;
+    padding: 2rem;
+    border: 2px solid var(--black);
+}
+
+.input {
```

Comment:
Add 

```css
margin: 0 auto;
```

This would align it horizontally in the center. The current one makes it aligned towards left.

Reply:
about which element you are talking about?